themes/docdock/layouts/partials/header.html

```diff
@@ -16,15 +16,18 @@
             <ul class="sc-uJMKN fRSLiB">
                 <li>What We Do</li>
             </ul>
-        </a><a class="sc-jqCOkK glTukf" href="https://nuevofoundation.org/get-involved">
+        </a>
+        <a class="sc-jqCOkK glTukf" href="https://nuevofoundation.org/get-involved">
             <ul class="sc-uJMKN fRSLiB">
-                <li>Get Involved</li>
+                <li>Support us</li>
```

Comment:
Unfortunately (or fortunately) there was already a change by Dan in master that fixed the issue a while ago, we just didn't see it because the master branch had not been deployed to prod yet. This section doesn't seem to be doing anything and instead the headers are being populated by lines 54-80. Thanks for looking into it! I would just abandon this PR.